pallets/runtime/src/runtime.rs

```diff
@@ -23,6 +23,7 @@ use polymesh_runtime_common::{
 };
 use polymesh_runtime_group as group;
 use polymesh_runtime_identity as identity;
+use polymesh_runtime_treasury as treasury;
```

Comment:
I recommend using `pallet_treasury` instead of `polymesh_runtime_treasury` name, I already change all the pallet's name to `pallet_xx` in my bootstrapping PR.

Reply:
I like the renaming. We have other pallets with `polymesh_runtime_xxx` so we could do it all of them in a small ticket.